deploy/docker/Dockerfile

```
@@ -13,9 +13,20 @@
 # limitations under the License.
 
 # Modified from https://github.com/rootfs/nfs-ganesha-docker by Huamin Chen
-ARG FEDORA_VERSION=35
 
-FROM fedora:${FEDORA_VERSION} AS build
+# List of Fedora versions: https://en.wikipedia.org/wiki/Fedora_version_history#Version_history
+#
+# FIXME: Fedora version 35 is currently in pre-beta and to be released fully
+#        2021-10-19 but required for arm support alongside amd64 and arm64
+#        support. When it is released:
+#
+#        1. Bump this version to 35
+#        2. Enable tests for --platform linux/arm
+#        3. Update /cloudbuild.sh to build linux/arm as well
+#
+ARG FEDORA_VERSION=34
```

Could you fix fedora version here please. The change was introduced by https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/pull/61

```suggestion
ARG FEDORA_VERSION=35
```

---

Yes I noted that version bump, but was it correct or a mistake? I assumed it was a mistake. There is a docker image named version 35, but fedora's latest release is 34 and the 35 release haven't yet become a beta release.

If I would bump to FEDORA_VERSION=35, it would in the pre-beta release state, I think it seem to break the amd64 functionality to use `microdnf` on the fedora-minimal container that starts to error.

---

Oh wait, is NFS Ganesha v3.5 mirroring Fedora versions? I've assumed it was a coincidence that they had the same version.

Is NFS Ganesha v3.5 required to be installed in Fedora 35?

---

@kvaps can you comment if you want Fedora 35 as introduced in #61, even though it isn't a stable release yet?

---

Hi @consideRatio!
I don't think if this is really matter. Let it use fedora 34 then. 🙂

---

Thanks! Going with fedora 34!